portal/code/company-portal/src/helpers/utils.ts

```diff
@@ -164,35 +164,36 @@ export function getClientRolesComposite(): Promise<string[]> {
   return promise;
 }
 
-export function uploadDocument(files:IFileWithMeta): Promise<any> {
+export function uploadDocument(file): Promise<any> {
 
   const token = UserService.getToken();
   const u = `${url}/${endpoint}/documents`;
-  console.log(typeof(files[0]));
   let formdata = new FormData();
-  formdata.append("document", files[0].file);
-  const promise = new Promise<any>((resolve, reject) => {
-    fetch(u, {
-      method: "POST",
-      headers: {
-        Authorization: `Bearer ${token}`,
-        // "Content-Type": "multipart/form-data",
-      },
-      body:formdata
-    })
-    .then((res) => res.text().then((data) => {
-      if (res.ok) {
-        resolve('Sent Invite');
-      } else {
-        reject(res.status);
-      }
-      }))
-      .catch((error) => {
-        // alert(error);
-        console.log(error, error.message, error.status);
-        reject(error.status);
-      }); 
-    });
-  return promise
+  formdata.append("document", file.file);
+  return fetch(u, {
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${token}`,
+      // "Content-Type": "multipart/form-data",
+    },
+    body:formdata
+  });
+
+  // const promise = new Promise<any>((resolve, reject) => {
```

Comment:
what's the purpose of keeping this comments?

Reply:
will remove this in next PR